game/hud/src/services/session/layout.ts

```diff
@@ -109,18 +121,80 @@ const initialState = {
   widgets: defaultWidgets(),
 }
 
-let getInitialState = () => {
-  var storedState = JSON.parse(localStorage.getItem(localStorageKey)) as LayoutState;
+interface Size {
+  width: number;
+  height: number;
+}
+
+const px2pcnt = (pos: Position, screen: Size) : Position => {
```

Comment:
Use function dec's instead of expresisons

https://github.com/airbnb/javascript#functions--declarations